src/com/flowers/kiev/Main.java

```diff
@@ -0,0 +1,38 @@
+package com.flowers.kiev;
+
+import com.flowers.kiev.entity.Flower;
+import com.flowers.kiev.service.FlowerSaver;
+import com.flowers.kiev.service.FlowerStore;
+import com.flowers.kiev.service.FlowersLoader;
+
+import java.util.stream.Stream;
+
+public class Main {
```

Comment:
Let's rename Main to ConsoleApplication or ApplicationRunner or ConsoleApplicationRunner ext.